src/phongo_bson.c

```diff
@@ -982,34 +896,147 @@ bool php_phongo_bson_data_to_zval(const unsigned char* data, int data_len, zval*
 	return retval;
 }
 
-/* Converts a BSON value to a ZVAL. */
-bool php_phongo_bson_value_to_zval(const bson_value_t* value, zval* zv)
+/** Converts a BSON value to a zval, returning BSON objects and arrays as
```

Comment:
We don't use doc blocks in PHPC.

```suggestion
/* Converts a BSON value to a zval, returning BSON objects and arrays as
```

Reply:
Good point, especially considering that C doesn't have the concept of docblocks at all.

That said, there are a number of other instances where we use docblocks which we may want to clean up at some point. I've created PHPC-2241 to track cleaning those up.